const/name-to-flag.yaml

```diff
@@ -115,13 +116,9 @@ Karyna: PL
 Kasandra: PL
 Kat Von Kaige: WALES
 Kevin Williams: SCOTLAND
-Kinga Miotke: PL
-Kinga Różańska: PL # Kinga Miotke == Kinga Różańska
```

Comment:
zrób nowy branch z maina, bo wątpię żebyś chciał usuwać

Reply:
Kinga ma już swoją teczkę, więc nie ma raczej potrzeby, żeby była obecna na liście imion - o ile dobrze rozumiem, to name-to-flag ma niższy priorytet, niż imiona/flagi zaciągane z indywidualnych wpisów ludzi.

Reply:
fakt, moje niedopatrzenie